### PR TITLE
retrun back the account to playbookdispatcher

### PR DESCRIPTION
**Pull request by mgold1234**
this field is mandatory for now

Signed-off-by: mgold1234 <mgold@redhat.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Review by AaronH88 — CHANGES_REQUESTED** [review]
populate account please

File: pkg/services/updates.go [hunk 1]
```diff
@@ -187,6 +187,7 @@ func (s *UpdateService) CreateUpdate(id uint) (*models.UpdateTransaction, error)
 		payloadDispatcher := playbookdispatcher.DispatcherPayload{
 			Recipient:   device.RHCClientID,
 			PlaybookURL: playbookURL,
+			Account:     update.Account,
```
Comment (ldjebran) [inline pkg/services/updates.go:190]: are we sure update has account with A VALUE != ""

Comment (AaronH88) [reply pkg/services/updates.go:190]: Update does not have account, you need to get account number from somewhere else, like context

Comment (ldjebran) [reply pkg/services/updates.go:190]: I suppose update is UpdateTransaction

Comment (ldjebran) [reply pkg/services/updates.go:190]: but if account is available in context , we can use context as a short cup , and do not touch anything else

**Review by de1987 — APPROVED** [review]

**Pull request merged**